packages/safe-tools-client/app/services/network.ts

```diff
@@ -0,0 +1,48 @@
+import Service from '@ember/service';
+
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+import {
+  getConstantByNetwork,
+  Networks,
+  networks,
+  supportedChainsArray,
+} from '@cardstack/cardpay-sdk';
+
+const DEFAULT_NETWORK = 'mainnet' as const; // TODO: add default based on env
+
+export default class Network extends Service {
+  @tracked chainId = getConstantByNetwork('chainId', DEFAULT_NETWORK);
+  @tracked name = getConstantByNetwork('name', DEFAULT_NETWORK);
+  @tracked symbol: Networks = DEFAULT_NETWORK;
```

Comment:
If these three values always change together, perhaps it would be better to have a single tracked `Network`-type value and convenience getters if desired.

Reply:
Ya, my first line of thought was that this.network is already the common denominator and that's why I was using Network as a type, but I'm ok with renaming it